network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html

```diff
@@ -0,0 +1,70 @@
+{% load primary_active_nav %}
+
+<div id="primary-nav-container">
```

Comment:
It feels like there is work we can probably do bringing down the level of div nesting here, but this is not the PR for that.

Reply:
_runs away from nav clean up work 😂_